src/mode/remapper.ts

```diff
@@ -120,8 +125,17 @@ class Remapper {
       }
 
       vimState.isCurrentlyPerformingRemapping = false;
-
       return true;
+    } else {
+      // Check to see if a remapping could potentially be applied when more keys are received
+      for (let remap of this._remappings) {
```

Comment:
Unless I am missing something obvious that you can see pretty clearly...

It is definitely possible, the problem I see is that we call handleKeyEvent, which calls sendKey, then it calls handleKeyEventHelper which uses the property

I could make a data structure that returned a few properties but I don't think it will end up being cleaner...

sendkey only returning a boolean seems to make sense to me, I would really only change this to return a true/false/maybe with the data structure anyways